views/doctor/patientOfDoctor.php

```diff
@@ -0,0 +1,172 @@
+<!DOCTYPE html>
+<html lang="en" dir="ltr">
+  <head>
+    <meta charset="utf-8">
+    <title>Patient Of Doctor</title>
+    <link rel="stylesheet" href="../../static/style.css">
+    <script type="text/javascript" href="../../static/register.js">
+    </script>
+  </head>
+  <body>
+
+    <section>
+    <?php
+    include ('../../common-functions.php');
+    check_session(3);
+    echo '<a href=' . "doctorHome.php" . '>' . 'Back' . '</a>';
+
+    $patient = $_GET['patient_id_num'];
+
+    $id = intval($_SESSION['id']);
+    $day = date('Y-m-d');
+
+    $db_link = mysqli_connect("localhost", "root", "", "retire");
+
+    if ($db_link == false) {
+      die("ERROR: Could not connect. " . mysqli_connect_error());
+    }
+
+    $sql = "SELECT Fname, Lname FROM users WHERE id LIKE $patient";
+
+
+    $result = mysqli_fetch_row(mysqli_query($db_link, $sql));
+
+
+    if (!empty($result[0]) AND !empty($result[1])) {
+
+      echo '<h2>' . 'Patient Of Doctor for ' . $result[0] . ' ' . $result[1] . '</h2>';
+
+    }
+
+    else {
+      die('Error could not get desired patient info:' . mysqli_connect_error());
+    }
+
+    $sql = "SELECT  a.day, p.morning_med, p.afternoon_med, p.night_med, p.comment
+           FROM appointments as a JOIN prescriptions as p on (a.patient_id = p.patient_id) AND (a.day = p.appt_day)
+           JOIN users as u ON (a.patient_id=u.id)
+           WHERE doctor_id LIKE $id AND a.patient_id LIKE '$patient'";
+
+
+           $result = mysqli_query($db_link, $sql);
+
+           if (empty($result)) {
+             die("<p class='error'>Patient info has no results</p>");
+           }
```

Comment:
Most of the rest of the file is indented from here; it should be in line with $sql